Basic_viewer/include/CGAL/Qt/Basic_viewer.h

```diff
@@ -11,8 +11,8 @@
 // Author(s)     : Guillaume Damiand <guillaume.damiand@liris.cnrs.fr>
 //                 Mostafa Ashraf <mostaphaashraf1996@gmail.com>
 
-#ifndef CGAL_BASIC_VIEWER_H
-#define CGAL_BASIC_VIEWER_H
+#ifndef CGAL_BASIC_VIEWER_QT_H
+#define CGAL_BASIC_VIEWER_QT_H
```

Comment:
If you not mind, our guidelines recommend `CGAL_QT_BASIC_VIEWER` (to follow the order of "words" in the header file path.